newswires/client/src/App.tsx

```diff
@@ -39,23 +39,34 @@ import { configToUrl, defaultQuery } from './urlState';
 const Alert = ({
 	title,
 	icon = 'warning',
-	children,
 }: {
 	title: string;
 	icon?: string;
-	children: React.ReactNode;
 }) => (
-	<EuiToast
-		title={title}
-		iconType={icon}
+	<div
 		css={css`
-			border-radius: 0;
-			background: #fdf6d8;
-			position: fixed;
+			.header-only span {
```

Comment:
The screenshots look great, but I feel a bit uneasy about using `!important` for styling here. Is there any reasonable way of avoiding it? If not, could we add an inline comment explaining why we think it's necessary?

Reply:
I removed the `!important` annotation and narrowed down the selector.